test/ConcurrentCacheTest.cpp

```diff
@@ -12,6 +12,7 @@
 #include "../../src/util/Cache.h"
 #include "../../src/util/ConcurrentCache.h"
 #include "../../src/util/Timer.h"
+#include "util/DefaultValueSizeGetter.h"
```

Comment:
Also fix the includes above.

Reply:
Done, see [f765e2c](https://github.com/ad-freiburg/qlever/pull/1111/commits/f765e2c675cf0b0bca4361f63dfc77c7e9a631c3).